### DISCUSSION TITLE
Is it possible to set `data-s` attributes on buttons for eg automation testing?

### DISCUSSION THREAD

**Discussion by sebm** [twilio-labs/paste#664] [Q&A] [answered]
I have a button.
```
					<Button
						variant="primary"
						loading={isLoading}
						onClick={foo}
					>
					 Foo
					</Button>

```

My e2e tests would like it if I could add data attributes to my button, for example
```
					<Button
						data={{ s: 'the-foo-button' }}
						variant="primary"
						loading={isLoading}
						onClick={foo}
					>
					 Foo
					</Button>
```

Is that possible?

**Comment by SiTaggart** [comment, marked as answer]
HI @sebm 

This has come up before, and i'm not sure where this syntax at Twilio has come from `data={{ s: 'the-foo-button' }}`, but you can just give it an html data attribute `data-s="the-foo-button"`.

Let us know if that works.

**Reply by sebm** [reply]
Thanks @SiTaggart, that does the trick!
